exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/QueryWrapper.java

```diff
@@ -83,12 +84,13 @@ public QueryResult run(final WorkManager workManager, final WebUserConnection we
     float usagePercent = getHeapUsage();
 
     // Wait until the query execution is complete or there is error submitting the query
-    logger.debug("Wait until the query execution is complete or there is error submitting the query");
+    if (logger.isDebugEnabled()) {
+      logger.debug("Wait until the query execution is complete or there is error submitting the query");
+    }
     do {
       try {
-        isComplete = webUserConnection.await(TimeUnit.SECONDS.toMillis(1)); /*periodically timeout to check heap*/
-      } catch (Exception e) { }
-
+        isComplete = webUserConnection.await/*timedWait*/(TimeUnit.SECONDS.toMillis(1)); //periodically timeout 1sec to check heap
```

Comment:
`/*timedWait*/` -> please remove this is obvious
`1sec` -> space

Reply:
My bad. That was actually a method I introduced before and reverted in the last commit.